src/main/scala/beam/sim/BeamHelper.scala

```diff
@@ -388,6 +389,25 @@ trait BeamHelper extends LazyLogging {
     }), scala.concurrent.duration.Duration.Inf)
   }
 
+  def writeScenarioPrivateVehicles(scenario: MutableScenario, beamServices: BeamServices) = {
+    val csvWriter: FileWriter = new FileWriter("householdVehicles.csv", true)
+    try {
+      csvWriter.write("vehicleId,vehicleType,householdId\n")
+      scenario.getHouseholds.getHouseholds.values.asScala.foreach { householdId =>
+        householdId.getVehicleIds.asScala.foreach { vehicle =>
+          beamServices.privateVehicles
+            .get(vehicle)
+            .map(
+              v => v.id.toString + "," + v.beamVehicleType.id.toString + "," + householdId.getId.toString + "\n"
+            )
+            .map((csvWriter.write(_)))
```

Comment:
Seems like you don't use return value and do side-effect, so you could consider to use `foreach` instead of `map`